### DISCUSSION TITLE
Discussion Forum/Community for imx8

### DISCUSSION THREAD

**Discussion by ghost** [zephyrproject-rtos/zephyr#61825] [General] [answered]
General Question for NXP projects: Is there a discussion forum/community for arm on imx8? The main repository for imx does not seem to have discussion site available.

Thanks!

**Comment by nordicjm** [comment]
Not sure about this specifically but there is a zephyr discord linked on https://github.com/zephyrproject-rtos/zephyr/#community-support which has an nxp group

**Reply by ghost** [reply]
Okay! Thank you so much!

**Comment by DerekSnell** [comment, marked as answer]
Hi @wcc-ccw ,
Just to provide more info for you and others, here are more support channel options for i.MX questions.  I am not sure what repo you are referring to when you stated:
> The main repository for imx does not seem to have discussion site available.

But for all Zephyr-related questions, including on i.MX SOCs, you can post your questions here (like you did) in the [Zephyr GitHub Discussion](https://github.com/zephyrproject-rtos/zephyr/discussions/new?category=q-a).  NXP monitors these Discussions, and this is our preferred location for Zephyr questions.  NXP also monitors the Zephyr-NXP Discord channel that nordicjm directed you to.

For other questions about the NXP SOCs that are not specific to Zephyr, you can search and post your questions at NXP's Community forum.  This link is the forum for [i.MX Processors](https://community.nxp.com/t5/i-MX-Forums/ct-p/imx-processors).

Best regards

**Reply by ghost** [reply]
The information is super helpful! Thanks!
